<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>458</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>458</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>458</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OIDN-GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSpinBox" name="spinBox_maxmem">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;approximate maximum amount of scratch memory to use in megabytes (actual memory usage may be higher)&lt;/p&gt;&lt;p&gt;limiting memory usage may cause slower denoising due to internally splitting the image into overlapping tiles, &lt;/p&gt;&lt;p&gt;but cannot cause the denoising to fail&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="suffix">
      <string> MB</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>512000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>6000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;approximate maximum amount of scratch memory to use in megabytes (actual memory usage may be higher)&lt;/p&gt;&lt;p&gt;limiting memory usage may cause slower denoising due to internally splitting the image into overlapping tiles, &lt;/p&gt;&lt;p&gt;but cannot cause the denoising to fail&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Max Memory:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[Options]</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_useaovs">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>181</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>improve the denoising quality, preserving more details in the image.</string>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>Use Albedo+Normal AOV</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_hdr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>161</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>improve the denoising quality, preserving more details in the image.</string>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>HDR (32/16bit float)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_compress">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>PXR24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Zip</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Zip (Scanline)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PIZ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RLE</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Compression:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_setArnoldAOV">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Arnold AOVs</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_setRedshiftAOV">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Redshift AOVs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>AOV name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Albedo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Normal</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_layernameNormal">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_layernameAlbedo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>albedo</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_outputPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>711</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_runDenoiser">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Denoiser</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_total">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>711</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>Total: %p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_animStart">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Animation Start: -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_animEnd">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>End: -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_animTotal">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Total: -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addSingleFile">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Single File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addSequenceFiles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Sequence Files</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_inputPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>711</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Input: -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stats">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
